packages/subgraph/scripts/import-plugin-repo.ts

```diff
@@ -7,23 +7,16 @@ const rootDir = path.join(__dirname, '../../../'); // Adjust the path as necessa
 dotenv.config({path: path.join(rootDir, '.env')});
 
 // Extract Repo address from the production-network-deployments.json
-function extractAndWriteAddressToTS(jsonPath: string): void {
-  // Read the production-network-deployments.json file
-  const aragonDeploymentsInfo = JSON.parse(fs.readFileSync(jsonPath, 'utf8'));
-
-  // Get the network from environment variables
+function writeAddressToTS(): void {
```

Comment:
here we need some big TODO...etc

Reply:
Added a comment: b03ef208b7e2d11b1e387568cbbd98dec656ac5d